README.md

```diff
@@ -0,0 +1,41 @@
+# envars
+
+Command-line Python application to manage environment variables and AWS KMS encrypted secrets in a single file.
+
+## Requirements
+
+Your AWS account should have a KMS key created with an alias of `alias/envars`.
```

Comment:
Can this be explained further or a link to a guide provided please? I need an ELI5 version haha

Reply:
link added